### ISSUE TITLE
The detections files on the nuscenes dataset

### ISSUE THREAD

**Issue by xiang-1208** [dvlab-research/LargeKernel3D#16]
Good job！Could you provide the detections files on the nuscenes dataset training set and the validation set?
